src/classes/RD_RecurringDonations_TDTM.cls

```diff
@@ -86,6 +86,11 @@ public class RD_RecurringDonations_TDTM extends TDTM_Runnable {
                      r.npe03__Open_Ended_Status__c != system.label.npe03.RecurringDonationClosedStatus)) {
                     r.addError(system.label.npe03.RecurringDonationTooManyInstallmentsError);
                 }            
+                // Reset this field to false if the installment period is not Monthly or Quarterly
```

Comment:
Right now I think the only intervals we support based on months are Monthly and Quarterly, both of which are supported by the new feature. To my knowledge there's no current way for a customer to create a custom interval, such as "Every 2 months". It only supports the specific defined Intervals - Weekly, Monthly, Quarterly, Yearly, and Every 1st & 15th.